applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>With a header like like "Plotting Chicago Bernard WBGT" at the top of this section, it will be separated on the "Content" side bar, would be helpful as a sub-header to Bernard</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>+1</p>